### ISSUE TITLE
[Bug]: macOS 15: Game paused if other full-screen apps is currently active

### ISSUE THREAD

**Issue by noni102** [PlayCover/PlayCover#1568]
### Describe the bug

game stops working if i turn my window to another desktop full screen 
actually i chose playcover rather than other ways to sideload apps because i need to let Arknights run itself all background
now it has to be the window active
but the new macOS seems to disagree

### Steps to reproduce

alors just update macOS developer beta 
and lunch a game then turn to a safari window full-screen
the game muted and paused, like the apps side loaded
but i only tried Arknignts, not sure if this is global or just Arknignts, or maybe just me

### Expected behaviour

game muted and paused

### Crash log

```shell
not really
```

### What version of PlayCover are you using?

3.0.0

### What version of macOS are you using?

macOS beta (please state the specific version)

### Issue Language

- [X] Yes my issue is written in English

**Comment by lslqtz** [comment]
I must point out that it also affects apps I download from the Mac App Store.

At the same time, the new version strengthens Gatekeeper verification. Users may have to use ```xattr -cr <Application Path>``` to solve the problem of not being able to open the app after installing it (perhaps when the user has not allowed any source).

**Comment by FlyMeToTheMoonAndLetMePlayAmongTheStars** [comment]
Personally I think this is a good feature and it's pretty useful if you want to leave a game open and not have it hog CPU/GPU resources. I've been using this so I won't have to open Genshin each time and go through the annoying start-up.

If any "fix" were to be implemented, it would be optimal to give the user the choice of whether games should automatically pause if it's full screen & not active. 

In the meantime, perhaps as a workaround to what you want to achieve, maybe try running Arknights in a window instead of fullscreen? Does it still pause if it's windowed?

**Comment by dmnbme** [comment]
I encountered this problem too, with Genshin Impact. But this is the only app I maximized. I think it's a feature instead of a bug because when i leave for a while, it seems to be reloading. I hate this feature.

**Comment by figurativly1984** [comment]
Still here with the release of Sequoia unfortunately. When I go from the game (windowed) to another desktop (with a windowed browser) the game just completely freezes. Hopefully a proper workaround can be found soon as it does make playing afk games very annoying.

**Comment by jhonchu5** [comment]
I encountered this issue after updating to macOS 15. The main feature I value in PlayCover is background auto-play. When I’m playing AFK and AFK 2, I need background auto-play to automatically progress through the game. Do you know when this feature will be available on macOS 15?

**Comment by Ivan-YihuaLi** [comment]
Same issue here. Just updated to MacOS 15 and game(windowed and maximized) freezes whenever I switch to apps in another desktop.

**Comment by shivendratrivedi99** [comment]
Still facing the issue on mac os stable Sequoia 15.0 release (24A335)

**Comment by danyadev** [comment]
It looks like I encountered this issue a long time ago when PlayCover was in beta, and the macOS version was 12 or 13...

I don't think that this information will help though, but I hope there is a way to fix this problem since it's making my gaming experience much worse. For example, if I'm in the middle of a storyline, I can't take a screenshot and send it to someone, search for needed information, or respond to messages in other apps, because there is a chance to disconnect me

For now it's better to rollback to macOS 14

**Comment by abdoufma** [comment]
Please take a look at this issue. it makes using PlayCover way more annoying that it needs to be.

**Comment by Catta1997** [comment]
> Please take a look at this issue. it makes using PlayCover way more annoying that it needs to be.

its not an issue but a macOS sequoia "feature", nothing we can do

**Comment by abdoufma** [comment]
This behavior does have exceptions though (like video players)

Can't Play over register itself as one of those apps?

**Comment by XQZ233** [comment]
If you don't mind the trouble, you can snap your MacBook and open it, and then you will find that Arknights will run in full screen.

**Comment by Catta1997** [comment]
> This behavior does have exceptions though (like video players)
> 
> Can't Play over register itself as one of those apps?

no, the problem is not PlayCover itself but the installed apps, the apps are catalyst app (catalyst is the name of the apps that is a porting of iOS apps) and now that type of app in macOS Sequoia work exactly like iOS so when an app is not active is like put in background an app in iOS so is frozen by the system

**Comment by juliusl** [comment]
I think I might have found a fix, right-click on the app icon from the tool, options, assign to all desktops
